lib/util/github/tags.ts

```diff
@@ -0,0 +1,24 @@
+import { logger } from '../../logger';
```

Comment:
please add tests to this folder so if we run on only this folder coverage should still be 100%

Reply:
existing coverage without my change in this dir:

```
=============================== Coverage summary ===============================
Statements   : 2.99% ( 987/32958 )
Branches     : 1.5% ( 163/10843 )
Functions    : 2.01% ( 81/4015 )
Lines        : 3.03% ( 984/32474 )
================================================================================
Jest: "global" coverage threshold for statements (100%) not met: 2.99%
Jest: "global" coverage threshold for branches (98%) not met: 1.5%
Jest: "global" coverage threshold for lines (100%) not met: 3.03%
Jest: "global" coverage threshold for functions (100%) not met: 2.01%
```

Added tests are 100% for `tags.ts`:
<img width="1721" alt="image" src="https://user-images.githubusercontent.com/4987015/218505737-18136be5-44e8-4910-9eed-6e7947f0a4a7.png">

Reply:
Confirmed locally that test coverage is 100% in each (independently)